google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -167,14 +167,17 @@ def run(self):
             item_counter = 0
             last_res_type = None
             with Inventory(self.session, self.inventory_id, True) as inventory:
-
+                root = inventory.get_root()
                 self.model.add_description(json.dumps({
                     "source":"inventory",
                     "source_info":str(inventory.index),
-                    "pristine":True
+                    "source_root":self._type_name(root),
+                    "pristine":True,
+                    "GSuite_enabled":inventory.type_exists(
```

Comment:
Can we use `gsuite_enabled` instead of `GSuite_enabled`? Or if there is any reason we need to capitalize this?

Reply:
gsuitte_enabled then